bundles/org.openhab.automation.jrubyscripting/README.md

```diff
@@ -0,0 +1,74 @@
+# JRuby Scripting 
+
+This add-on provides [JRuby](https://www.jruby.org/) 9.3.1 that can be used as a scripting language within automation rules.
+
+## JRuby Scripting Configuration
+
+JRuby configuration parameters may be set by creating a jruby.cfg file in $OPENHAB_CONF/services/
```

Comment:
Is there a reason why it is only possible to configure this using a file and not using the UI? 

To make this configurable using the UI you can add a `ConfigurableService` like the openhabcloud [CloudService](https://github.com/openhab/openhab-addons/blob/c79fd062569221451e6809c4afd52ecbac9822a9/bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudService.java#L67-L71).  Such services get their configuration using config descriptions, very similar to bindings, see [config.xml](https://github.com/openhab/openhab-addons/blob/c79fd062569221451e6809c4afd52ecbac9822a9/bundles/org.openhab.io.openhabcloud/src/main/resources/OH-INF/config/config.xml).

Reply:
Is it possible to have default values that reference directories ($OPENHAB_CONF) in the UI? Or do we say what the default is in the description?  If we do the UI for this, do we get an empty string if they don't fill anything out or do we get ""?

Reply:
It probably won't expand the variables in paths automatically, but you could make it an optional parameter and explain in the description how it behaves when it is left empty. That is the approach used for this in the [upnpcontrol binding](https://github.com/openhab/openhab-addons/blob/2a8e9b6e938479cfeb21a4ebd0251a6207c86030/bundles/org.openhab.binding.upnpcontrol/src/main/resources/OH-INF/binding/binding.xml#L9-L15).

Another approach is to expand the variable yourself which seems to be done in the [logreader binding](https://github.com/openhab/openhab-addons/blob/c79fd062569221451e6809c4afd52ecbac9822a9/bundles/org.openhab.binding.logreader/src/main/resources/OH-INF/thing/reader.xml#L27-L31).

I don't have a strong preference for one implementation over the other as long as it can be configured using the UI.

It might be a good idea to add support for expanding environment variables in strings in openhab-core one day though so it can be used anywhere without having to duplicate the code for this. :slightly_smiling_face: